code/Test_definitions/carrier-billing-retrievePayment.feature

```diff
@@ -0,0 +1,120 @@
+Feature: CAMARA Carrier Billing API, v0.3 - Operation retrievePayment
+  # Input to be provided by the implementation to the tester
+  #
+  # Implementation indications:
+  # * 
+  #
+  # Testing assets:
+  # * A phone number eligible for payment (no restrictions for it to be used to perform a payment)
+  # * Several payments performed in different status in the environment (at least 10)
```

Comment:
not necessary no ? just one is good enough?

Reply:
The comment is to have several payemenst in different status so as you can check different status are returned